deps.edn

```diff
@@ -66,7 +66,9 @@
   {:main-opts   ["-m" "zero.main"]}
 
   :test
-  {:extra-deps  {lambdaisland/kaocha {:mvn/version "1.0.632"}}
+  {:extra-deps  {lambdaisland/kaocha {:mvn/version "1.0.632"}
+                 org.liquibase/liquibase-cdi {:mvn/version "3.8.5"}
+                 org.liquibase/liquibase-core {:mvn/version "3.8.5"}}
```

Comment:
yeah, the liquibase log fetish is pretty annoying. goolging around there are people takling [about this too](https://stackoverflow.com/questions/20880783/how-to-get-liquibase-to-log-using-slf4j). Since it does not affect the test/code as you mentioned, I plan to keep this checked in and move to use liquibase command line if it starts to bite us, wdyt?

(I really cant't make myself like liquibase given what we have experienced these days tbh 🤦 )

Reply:
I think that's totally alright, just wanted to flag that the warning is basically expected behavior if we bring in dependencies that happen to implement SLF4J's interface.

Reply:
I also tried: 
```org.liquibase/liquibase-cdi {:mvn/version "3.8.5"  :exclusions [ch.qos.logback/logback-classic]```
But it seem liquibase has its fetish into logback that it uses logback in the constructor... So ultimately I think we could try run liquibase from commandline..

Reply:
I will try it again, it seems others also tried this https://github.com/metabase/metabase/blob/edf1c0366cfa754cb929b20d00256ff4a6a36fa9/project.clj#L119

Reply:
OK I give up, I'm not sure why https://github.com/metabase/metabase/commit/4968993e13575042de6cb7f204002673f63bd4cb works for them but causes errors for us. (maybe lein does better than deps.edn?)